packages/plugin-ext/src/main/browser/view/tree-views-main.ts

```diff
@@ -197,7 +194,6 @@ export class TreeViewsMainImpl implements TreeViewsMain, Disposable {
         }));
 
         this.toDispose.push(treeViewWidget.model.onSelectionChanged(event => {
-            this.contextKeys.view.set(treeViewId);
```

Comment:
Would it make sense to remove the `view` context key? It never makes sense to set it in the global context, as I understand it.

Reply:
I removed this context key, as it indeed makes no sense on a global context. _focusedView_ makes sense here, but not _view_.